marketplace/netlify-build/src/index.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { init } from 'contentful-ui-extensions-sdk';
+import { Select, Option, Button, Icon } from '@contentful/forma-36-react-components';
+import NeflifySideBarBuildButton from './NetlifySideBarBuildButton';
+
+import styles from './styles';
+
+const readCsvParam = csv =>
+  (csv || '')
+    .split(',')
+    .map(val => val.trim())
+    .filter(val => val.length > 0);
+
+function createSiteStructure(parameters = {}) {
+  const buildHooks = readCsvParam(parameters.buildHookUrls);
+  const siteIds = readCsvParam(parameters.netlifySiteIds);
+  const names = readCsvParam(parameters.names);
+  const urls = readCsvParam(parameters.netlifySiteUrls);
+
+  return buildHooks.reduce(
+    (acc, _, i) =>
+      acc.concat([
+        {
+          buildHookUrl: buildHooks[i],
+          channel: buildHooks[i].split('/').pop(),
```

Comment:
we could add a comment for what we do here

Reply:
Added.